pages/set/_.vue

```diff
@@ -184,6 +184,9 @@
       displayDescription() {
         return langMapValueForLocale(this.set.description, this.$i18n.locale);
       },
+      enableRecommendations() {
```

Comment:
Add a check for the status of this in the mounted hook and the set.total watcher so as to prevent retrieval of recommendations.

Reply:
I have moved it to the getRecommendations method itself, where it checks for both this toggle and the logged in status.